csrankings-m.csv

```diff
@@ -232,6 +232,7 @@ Manjunath V. Joshi,DAIICT,http://www.daiict.ac.in/daiict/people/faculty.html,iDq
 Manmohan Chandraker,Univ. of California - San Diego,http://cseweb.ucsd.edu/~mkchandraker,oPFCNk4AAAAJ
 Manmohan Krishna Chandraker,Univ. of California - San Diego,http://cseweb.ucsd.edu/~mkchandraker,oPFCNk4AAAAJ
 Manohar Kaul,IIT Hyderabad,http://www.iith.ac.in/~mkaul,jNroyK4AAAAJ
+Manoj Gopalkrishnan, IIT Bombay,https://www.ee.iitb.ac.in/~manojg/,KqkkRVgAAAAJ

```

Comment:
Remove spaces after commas.

Reply:
EE, not CS. Needs justification for why they meet inclusion criteria.

Reply:
He is currently advising CS students :  [Sabyasachi Ghosh](https://sghosh767.wordpress.com/), [Ashwin Paul](https://github.com/aswinpaul)